### DISCUSSION TITLE
Daily 20/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#213] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by MarielaGomez** [comment]
Durante estos días estuvimos en comunicación con compañeros @yan0106 @Valentino399 @dfg259 que llevan el mismo ritmo en el proceso de aprendizaje, tiempo y recursos, auxiliamos a compañeros que por ciertas dificultades exentas a plantearlas aquí se han visto obstaculizadas, no llevamos el mismo tiempo que el resto del equipo lo que se dificulta estar al mismo nivel, seguimos avanzando en las tareas que tenemos asignadas y poniendo en practica con proyectos paralelos, no vamos a emitir cambios hasta tanto no estar seguros, se intentará conseguir los cambios pertinentes. También ayer tuve clases de Inglés donde participe para la ante ultima evidencia.

**Comment by BraianTroncoso** [comment]
Anoche finalicé mis tareas asignadas y también fuí realizando las que no se habían terminado que fueron asignadas hace 2 semanas o mas, lamentablemente la falta de comunicación y la poca o nula respuesta de mensajes por parte del equipo(se puede ver en contribuciones o la participación de las dailys) se complicó demasiado porque no fué fácil aprender y real izarlo al mismo tiempo pero dedicándole mas horas( dejando días de trabajo, perdiendo de ver a familiares ) pudimos los pocos que participamos avanzar para poder cumplir con lo requerido por el sprint actual. Cabe acotar que siempre fué ofrecida la ayuda e instado a reunirse en múltiples oportunidades pero como dije anteriormente nunca hubó respuesta, salvo los últimos días, como siempre.

**Comment by NataliaAlvarezIspc** [comment]
Ayer, finalmente pude hacer funcionar la registración, quedó subida al repo. También participe de la clase sincrónica del instituto.
Hoy intenté hacer que busque el mail en la base de datos, pero no pude comprobarlo ya que los nuevos cambios me generaron conflicto en la importación de librerías y no pude avanzar, dejé todo como estaba. Prácticamente fue un día perdido. Voy a seguir con otras tareas del front en lo que queda del día.
La verdad que la falta de respuesta y de avances por parte de la mayoría de los miembros del equipoo es una gran dificultad y genera muchísimos inconvenientes para los pocos que queremos seguir avanzando en el proyecto. Esto se hablo muchas veces, pero se hizo caso omiso. Lo cierto es que queda mucho por hacer y pocas manos para realizarlo.

**Comment by marceloedelgado** [comment]
Ayer estuve trabajando para hacer funcionar la tarjeta ampliada sin exito alguno. Entre trabajos y responsabilidades personales no pude avanzar porque ya a la tarde estuve en el encuentro de la clase con las profesoras Viviana y Anahí quienes vieron nuestra aplicación y nos dieron una devolución.

Hoy recien por la tarde pude hacer funcionar la implementación de la funcionalidad de la tarjeta ampliada cuando se toca sobre el boton "Consultar" del catalogo_activity hacia la tarjeta ampliada.

Estuve lidiando con la implementación y la curva de aprendizaje de este trayecto en el ISPC. 

Si bien todos estudiamos en el mismo instituto y la misma carrera, no por ello tenemos la misma realidad, o disponemos de la misma cantidad de tiempo para avanzar en el aprendizaje, cada uno atraviesa de manera individual dificultades para poder aprender y no es intencional sino que es la vida misma. Cada cual esta atravesado por una realidad diferente y en ese cruzamiento de realidades nos vemos aquí todos juntos intentando aprender y meter codigo, algunos mas otros menos pero nada invalida el proceso de aprendizaje no hay que olvidarse de eso. Estoy contento de poder haber hecho funcionar la funcionalidad de la tarjeta ampliada sin ayuda de mis compañeros a los cuales les pedi ayuda el sabado pasado para poder implementarlo y no tuve respuestas. Seguire adelante codeando mis tareas asignadas hasta el cierre del sprint.
